crypto/build.gradle

```diff
@@ -9,7 +9,7 @@ dependencies {
   compile 'com.google.guava:guava'
   compile 'com.github.jnr:jnr-ffi'
   compileOnly 'org.bouncycastle:bcprov-jdk15on'
-
+  compile 'org.miracl.milagro.amcl:milagro-crypto-java'
```

Comment:
we try to keep dependencies grouped and in alphabetical order. Mind moving this a few lines up?

Reply:
oh also can we make it compileOnly - and then no need to move it. We want to make it possible for folks to use only the SECP256k1 variant for example.

Reply:
I'm bad abc's, but I'll try.

Reply:
needs to be compileOnly